experimental/sql/dialects/ibis_dialect.py

```diff
@@ -38,7 +38,7 @@ class DataType(ParametrizedAttribute):
 
 
 @irdl_attr_definition
-class int32(DataType):
+class Int32(DataType):
```

Comment:
I think that this should be in a dedicated mini-PR (or at least a dedicated commit).